lib/media-queries.js

```diff
@@ -0,0 +1,43 @@
+
+var _ = require('lodash')
+var postcss = require('postcss')
+var rules = require('./rules')
+var selectors = require('./selectors')
+var declarations = require('./declarations')
+
+module.exports = function (root, opts) {
+
+  var result = {
+    total: 0,
+    unique: 0,
+    values: [],
+    contents: [] // Not sure what to name this
```

Comment:
Yeah, I originally had it as an object, but thought it might be useful to have each block listed separately. Mostly just exploring how to handle the media queries at this point.